pom.xml

```diff
@@ -32,45 +34,9 @@
 					</dependency>
 				</dependencies>
 			</plugin>
-			<plugin>
-				<groupId>org.jenkins-ci.tools</groupId>
-				<artifactId>maven-hpi-plugin</artifactId>
-				<version>1.106</version>
-				<configuration>
-					<pluginFirstClassLoader>true</pluginFirstClassLoader>
```

Comment:
Doesn't this change classloading in major way? Are you sure it's safe to remove?

Reply:
Not sure. Asking for review about this. 

It was added here: https://github.com/jenkinsci/semantic-versioning-plugin/commit/73977ace5bbaf2b79d0a8feabbbc2ba038b2f8c7#diff-600376dffeb79835ede4a0b285078036

****

According to the docs, I believe it is not needed here:

### Reference:

* https://wiki.jenkins.io/display/JENKINS/Plugin+Structure
* http://javadoc.jenkins-ci.org/hudson/PluginFirstClassLoader.html